frontend/src/scenes/trends/persons-modal/PersonsModal.tsx

```diff
@@ -85,6 +86,18 @@ export function PersonsModal({
 
     const totalActorsCount = missingActorsCount + actors.length
 
+    const getTitle = useCallback(() => {
+        if (typeof title === 'function') {
+            return title(capitalizeFirstLetter(actorLabel.plural))
+        }
+
+        if (title === BREAKDOWN_OTHER_STRING_LABEL) {
+            return 'Other'
+        }
```

Comment:
What if it's the `BREAKDOWN_OTHER_NUMBER_LABEL` ? We have somewhere a `isOtherBreakdown` function that'd be perfect here.

Personally I wouldn't have bothered with a `useCallback` here, and just inlined this into the body of the component. I doubt there are any measurable performance wins here. However to be truly accurate, you'd need to add `actorLabel.plural` also to the deps array.